### PR TITLE
Add auxiliary address space support to ZIS

### PR DESCRIPTION
**Pull request by ifakhrutdinov**
See [README.md](https://github.com/ifakhrutdinov/zss/blob/feature/zss-aux/zis-aux/README.md) for details.

### CODE REVIEW

File: zis-aux/test/utils/build/build.sh [hunk 1]
```diff
@@ -20,21 +20,21 @@ xlc "-Wa,goff" \
...
 "-Wl,ac=1" -I $COMMON/h -I $ZSSAUX/src -I $ZSSAUX/include \
 -o ../bin/shrmem64-target-test \
 $ZSSAUX/test/utils/src/shrmem64-test.c \
 $COMMON/c/alloc.c \
+$COMMON/c/shrmem64.c \
 $COMMON/c/utils.c \
 $COMMON/c/timeutls.c \
-$COMMON/c/zos.c \
-$ZSSAUX/src/shrmem64.c ; extattr +a ../bin/shrmem64-target-test
+$COMMON/c/zos.c ; extattr +a ../bin/shrmem64-target-test
```
Comment (daveyc) [inline zis-aux/test/utils/build/build.sh]: Should the ```extattr``` command be on it's own line?

Comment (ifakhrutdinov) [reply zis-aux/test/utils/build/build.sh]: There is ```;``` (command separator) between ```xlc``` and  ```extattr```.

Comment (daveyc) [reply zis-aux/test/utils/build/build.sh]: Yes, but it's off the radar for my tired old eyes

**Comment by daveyc** [comment]
Looks good

**Comment by ifakhrutdinov** [comment]
Reviewed the ASID check internally.

**Pull request merged**